tox.ini

```diff
@@ -9,6 +9,7 @@ python =
 
 [testenv]
 deps =
+    ordered-set
```

Comment:
This is not needed since all install requirements are automatically added to the test environment. You will have to run `tox -e py38 --recreate`, though, after making dependency changes.

```suggestion
```